src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/GATKReadFilterPluginArgumentCollection.java

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.hellbender.cmdline.argumentcollections;
+
+import java.io.Serializable;
+import java.util.Collection;
+
+/**
+ * An abstract ArgumentCollection for specifying parameters for {@link org.broadinstitute.hellbender.cmdline.GATKPlugin.GATKReadFilterPluginDescriptor}.
```

Comment:
I think we need more javadoc here. At a  minimum something like, "An abstract ArgumentCollection for defining the set of read filter descriptor plugin arguments that are exposed to the user on the command line. Subclasses should provide @ Argument annotations for the arguments that should be exposed."